Oracle/pcornet_mapping.csv

```diff
@@ -7,12 +7,8 @@ pcori_path,local_path
 \PCORI_MOD\RX_BASIS\PR\01\,\Medication\Outpatient\
 \PCORI_MOD\RX_BASIS\PR\02\,\Medication\PRN Inpatient Order\
 \PCORI_MOD\PDX\P\,\Diagnosis\\modifier_dx\billing_diagnosis\discharge\discharge_primary\
-\PCORI_MOD\PDX\P\,\Diagnosis\Billing\Primary\
```

Comment:
We do not want to remove mappings to `PDX` to `DX_SOURCE`.  Those mappings are needed for other portions of the transform.